docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,138 @@
+</head>
+```
+
+This will add the `carto` and the `mapboxgl` objects to the global namespace.
+
+> Note: Not every Mapbox GL version is compatible with CARTO VL. We highly recommend the following combination. If you are importing CARTO VL from npm, you have to use our Mapbox GL fork. Read more about how to do this in [the advanced guide]((https://carto.com/developers/carto-vl/guides/advanced))
+
+Create a `div` where the map is going to be drawn.
+
+```html
+  <div id="map"></div>
+```
+
+Style the `div` to ensure it will be displayed correctly.
+
+```css
+#map {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+}
+```
+
+#### Defining the basemap
+
+Once we have the `div`, we use the `mapboxgl` object to initialize our map using the following parameters:
+
+- **`container`** indicates where the map is going to be placed
+- **`style`** contains the information about the basemap
+- **`center`** indicates the area of the world we are going to visualize
+- **`zoom`** defines the default zoom level
+- **`dragRotate`** disables the map rotation
+
+```js
+const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [0, 30],
+      zoom: 2,
+      dragRotate: false
+    });
+```
+
+For basemaps you can add [Mapbox custom styles](https://www.mapbox.com/mapbox-gl-js/style-spec/) or choose one of the three predefined styles offered by CARTO:
+
+- **Voyager:** https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json
+- **Positron:** https://basemaps.cartocdn.com/gl/positron-gl-style/style.json
+- **Dark Matter:** https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json
+
+At this point you will have a basic map:
+
+[LiveExample](http://carto.com/developers/carto-vl/examples/maps/guides/getting-started/basic-set-up.html)
+
+### Defining the source
+
+In order to render data from CARTO you need to create a CARTO account and then get the necessary [credentials](https://carto.com/developers/fundamentals/authorization/).
+
+The first thing you need to do is [authenticate the client](https://carto.com/developers/carto-vl/reference/#cartosetdefaultauth) with your `user` and `apiKey`.
+
+```js
+carto.setDefaultAuth({
+    user: 'cartovl',
+    apiKey: 'default_public'
+});
+```
+
+The next step is to define the [`source`](https://carto.com/developers/carto-vl/reference/#cartosourcedataset) from your account to be displayed on the map. In the example below we are defining the `source` for a dataset named `populated_places` with all the populated places around the world.
+
+```js
+const source = new carto.source.Dataset('populated_places');
+```
+
+### Defining the Viz object
+
+A [`Viz object`](https://carto.com/developers/carto-vl/reference/#cartoviz) is one of the core elements of CARTO VL. It defines how the data will be styled, displayed, and processed. In this case we create an empty Viz object, that will use the default style.
```

Comment:
"... the default style (a red circle)"

Reply:
This is not 100% right, I will think a better explanation of what default does, but I do not think it is needed at this point.

Reply:
ok, then let them see the default style alive 😄